hypothesis-python/setup.py

```diff
@@ -85,6 +85,7 @@ def local_file(name):
     author_email='david@drmaciver.com',
     packages=setuptools.find_packages(SOURCE),
     package_dir={'': SOURCE},
+    # package_data={'': ['py.typed']},  # un-comment to release type hints
```

Comment:
Is this comment right? You also need to add the file back in to do that, don't you?

Reply:
I've actually left the file in - it just doesn't get included in the published wheels.

Reply:
Ah! For some reason I thought you hadn't. My mistake, sorry.

Reply:
No worries :smile: